### ISSUE TITLE
how to get time to do some other task

### ISSUE THREAD

**Issue by Gardezi1** [chfritz/meteor-easycron#5]
Hello I would like to know how can I get the time.Right now I'm doing like this but I can't get the time 
```
var recurrence = new Cron(everyMinute , {});

var everyMinute = new Cron(function() {
    console.log("another minute has passed!");
}, {});

```

I want to get the time rather then showing it so how would I do that ??

**Comment by chfritz** [comment]
sorry, I don't understand the question. Can you rephrase or show an example of what you would like to see happening? Also, your code example doesn't make sense to me. You can't pass a cron as the function to another cron.

**Comment by chfritz** [comment]
ps: you may want to ask questions on stackoverflow instead (where I'll see them, too, if you use the meteor tag). What you are asking does not seem to be an issue/bug with the code, so I'd like to close this issue.

**Comment by Gardezi1** [comment]
I'm sorry I created this bug but yeah there is not issue with the code

**Comment by chfritz** [comment]
no worries. I'd still like to try and help if I can (here or on stackoverflow). What is it you'd like to do?

**Comment by Gardezi1** [comment]
sorry for replying late.I need to know why am I getting this error ??

>Exception in setTimeout callback: TypeError: undefined is not a function
>I20151123-16:27:00.984(5)?     at packages/chfritz_easycron/packages/chfritz_easycron.js:99:1
>I20151123-16:27:00.984(5)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
>I20151123-16:27:00.984(5)?     at packages/meteor/timers.js:6:1
>I20151123-16:27:00.984(5)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:110:1)

This is the code 

```
var recurrence = new Cron(everyMinute , {});

var everyMinute = new Cron(function() {
    console.log("another minute has passed!");
    console.log("Let's Check");
        var _date = (new Date).toLocaleString().split(",");
        console.log(_date);
        var p = Reminders.findOne({_id: ServerSession.get('currentId')}).date.toLocaleString().split(",")[0];
        if (date == p){
        	p = Reminders.findOne({_id: ServerSession.get('currentId')}).time.toLocaleString();
        	tt = (new Date).getHours().toLocaleString();
        	s = (new Date).getMinutes();
        	if (s/10 == 0 ){
        		tt = tt + ":0" + s;
        	}else{
        		tt = tt+":"+s;
        	}

        	if (p ==  tt){
        		Router.go('alertPage')
        	}

        }
        console.log("Checking Done");
}, {});
```

I am checking if the time saved is equal to time stored in session (I am using matteodem/meteor-server-session for session so that I can access it on the server side) but it is throwing error which I mentioned above

**Comment by Gardezi1** [comment]
I solved the issue. but now I am getting this exception. Can you tell me why am I getting this error

>I20151123-18:09:00.189(5)? Exception in setTimeout callback: TypeError: undefined is not a function
>I20151123-18:09:00.190(5)?     at packages/chfritz_easycron/packages/chfritz_easycron.js:99:1
>I20151123-18:09:00.190(5)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
>I20151123-18:09:00.190(5)?     at packages/meteor/timers.js:6:1
>I20151123-18:09:00.190(5)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:110:1)
>I20151123-18:09:00.190(5)? another minute has passed!

**Comment by chfritz** [comment]
as I said `var recurrence = new Cron(everyMinute , {});` doesn't make any sense if `everyMinute` is itself a cron.

**Comment by Gardezi1** [comment]
can you please elaborate it to me since this is my first time using crone

**Comment by Gardezi1** [comment]
I kind of need to do this that when ever a user makes a new reminder a cron job should be made for that reminder. In short when ever a user makes a new reminder a cron  job should be initiated for that reminder
